schema-registry-examples/README.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+This module has applications to demonstrate how pravega applications can work with schema registry for different 
+serialization formats. 
+
+### Avro
+Avro package has two applications, namely AvroDemo.java and a Message bus sample application.
```

Comment:
nit: maybe you could use `this format` or add a link to the file for all the "XXX.java" files mentioned here.